packages/starlight/integrations/asides.ts

```diff
@@ -50,19 +65,11 @@ function s(el: string, attrs: Properties = {}, children: any[] = []): P {
  * </Aside>
  * ```
  */
-function remarkAsides(): Plugin<[], Root> {
+function remarkAsides(config: StarlightConfig, useTranslations: any): Plugin<[], Root> {
```

Comment:
Instead of `any` we can type `useTranslations`.

1. Import the type:

    ```js
    import type { createTranslationSystemFromFs } from '../utils/translations-fs';
    ```
    
2. Use it like:

   ```suggestion
   function remarkAsides(
   	config: StarlightConfig,
   	useTranslations: ReturnType<typeof createTranslationSystemFromFs>
   ): Plugin<[], Root> {
   ```